clj-kondo-resources/clj-kondo.exports/io.pedestal/pedestal.log/config.edn

```diff
@@ -0,0 +1,18 @@
+; Copyright 2022 Cognitect, Inc.
```

Comment:
Should this be 2023?

Reply:
I copied it from one of the existing files and didn't catch this :( . Will wait for @hlship to tell me what the right value should be, or if I can just leave it as-is, before making a change.

Reply:
(c) 2024 Nubank NA